### PR TITLE
fix: Name and tag of race image in docker repository should be different than production one

### PR DESCRIPTION
**Pull request by Matovidlo**
**Changes:**
- Name and tag of race image in docker repository should be different than production one.

-----------

### CODE REVIEW

**Review by hosekpeter — APPROVED** [review]
LGTM

**Comment by kacurez** [comment]
toto uz teda nebudes potrebovat nasadit ci?  https://keboolaglobal.slack.com/archives/C055K9CRFDX/p1733849063356059?thread_ts=1733834074.491109&cid=C055K9CRFDX

**Comment by Matovidlo** [comment]
@kacurez neviem ci to pomoze ci nie, niekde sa zvysil memory consumption a neviem teda ci je to imagom (skor nie) alebo kodom.

**Comment by kacurez** [comment]
pozeram na to a je vidiet ze je nasadeny image s tym racerom:
Zbuildoval sa s sha `6df78715eb1c48d5b...`
image
a [vidim](https://app.datadoghq.eu/container-images?query=stream%20AND%20image_name%3Aus-central1-docker.pkg.dev%2Fkeboola-prod-artifacts%2Fkeboola-as-code%2Fstream-service&inspect=us-central1-docker.pkg.dev%2Fkeboola-prod-artifacts%2Fkeboola-as-code%2Fstream-service%40sha256%3A6df78715eb1c48d5b6be746b80337699929be9a8ce922ce033794ae2b1f7acc6&panelTab=containers&view=explorer) ze je nasadeny 
image


Co je ale zvlastne ze napr v GCP [repu](https://console.cloud.google.com/artifacts/docker/keboola-prod-artifacts/us-central1/keboola-as-code?project=keboola-prod-artifacts) vobec nevidim ten `-race` image iba `stream-service`
image

**Review by kacurez — APPROVED** [review]
nie som si isty ci to pomozme ale mozme skusit, je vidiet ze sa nasadil image s tym race 
 ale pod prod tagom https://keboolaglobal.slack.com/archives/C055K9CRFDX/p1733913074422449?thread_ts=1733834074.491109&cid=C055K9CRFDX

File: .github/actions/push-image-aws-azure-gcp/action.yml [hunk 1]
```diff
@@ -55,14 +55,9 @@ runs:
...
       shell: bash
-      env:
-        IMAGE: ${{ inputs.imageName }}
-        TAG: ${{ inputs.imageTag }}
-        REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        REPOSITORY: ${{ inputs.ecrRepository }}
       run: |
-        docker tag $IMAGE:$TAG $REGISTRY/$REPOSITORY:$TAG
-        docker push $REGISTRY/$REPOSITORY:$TAG
+        docker tag ${{ inputs.imageName }}:${{ inputs.imageTag }} ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ecrRepository }}:$TAG
```
Comment (wokoman) [inline .github/actions/push-image-aws-azure-gcp/action.yml:59]:
```suggestion
        docker tag ${{ inputs.imageName }}:${{ inputs.imageTag }} ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ecrRepository }}:${{ inputs.imageTag }}
```

Plus i ty dva dole 🙈

**Review by wokoman — CHANGES_REQUESTED** [review]

File: .github/actions/push-image-aws-azure-gcp/action.yml (hunk 1 above)
Comment (Matovidlo) [reply .github/actions/push-image-aws-azure-gcp/action.yml:59]: dik

Comment (Matovidlo) [reply .github/actions/push-image-aws-azure-gcp/action.yml]: https://github.com/keboola/keboola-as-code/pull/2179/commits/cfcb37b26d64bb208c10b388b0dd8405cda7329c

**Review by wokoman — APPROVED** [review]
Fabulozni, dik!

**Pull request merged**